hash_practice/exercises.py

```diff
@@ -1,20 +1,53 @@
 
+import numbers

```

Comment:
Looks like this `import` statement never got used. I've seen this happen a few times across submissions and from my guess, your IDE added this import when you accidentally autocompleted at some point.

Not a major issue, but it's usually a good style habit to make sure to remove any unneeded `import` statements to keep the code cleaner. (Well, really, remove any unneeded code in general...)